be/src/exec/pipeline/hdfs_chunk_source.cpp

```diff
@@ -345,7 +345,7 @@ Status HdfsChunkSource::buffer_next_batch_chunks_blocking_for_workgroup(size_t b
             SCOPED_RAW_TIMER(&time_spent);
 
             ChunkPtr chunk;
-            _status = _read_chunk_from_storage(_runtime_state, &chunk);
```

Comment:
this code should not been try catched now, it may use the expr framework and some code of storage layer